src/argilla/client/feedback/dataset/mixins.py

```diff
@@ -379,3 +382,380 @@ def unify_responses(
+            if system_prompt
+            else "This is a conversational dataset that contains contexts. Please write the response to the context in the response field.",
+        )
+
+    @classmethod
+    def for_retrieval_augmented_generation(
+        cls: Type["FeedbackDataset"], number_of_documents: int = 1, use_markdown: bool = False
+    ) -> "FeedbackDataset":
+        """
+        You can use this method to create a basic dataset for retrieval augmented generation tasks.
+        To add items to your dataset, use the "add_item" method.
+
+        Args:
+            None
+
+        Returns:
+            A `FeedbackDataset` object for retrieval augmented generation containing "query" and "retrieved_document" fields and a TextQuestion named "response"
+        """
+        document_fields = [
+            TextField(
+                name="retrieved_document_" + str(doc + 1),
+                title="Retrieved Document " + str(doc + 1),
+                use_markdown=use_markdown,
+            )
+            for doc in range(number_of_documents)
+        ]
+        return cls(
+            fields=[TextField(name="query", use_markdown=use_markdown)] + document_fields,
+            questions=[
+                TextQuestion(name="response", description="Write the response to the query.", use_markdown=use_markdown)
+            ],
+            guidelines="This is a retrieval augmented generation dataset that contains queries and retrieved documents. Please write the response to the query in the response field by using the retrieved document.",
+        )
+
+    @classmethod
+    def for_sentence_similarity(
+        cls: Type["FeedbackDataset"], rank_similarity: bool = False, use_markdown: bool = False
+    ) -> "FeedbackDataset":
+        """
+        You can use this method to create a basic dataset for sentence similarity tasks.
+        To add items to your dataset, use the "add_item" method.
+
+        Args:
+            None
+
+        Returns:
+            A `FeedbackDataset` object for sentence similarity containing "sentence1" and "sentence2" fields and a RatingQuestion named "similarity"
+        """
+        return cls(
+            fields=[
+                TextField(name="sentence1", use_markdown=use_markdown),
+                TextField(name="sentence2", use_markdown=use_markdown),
+            ],
+            questions=[
+                LabelQuestion(
+                    name="similarity",
+                    labels=["not similar", "similar"],
+                    description="Choose the similarity between the two sentences.",
+                ),
+                RatingQuestion(
+                    name="similarity",
+                    values=[1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                    description="Rate the similarity between the two sentences.",
+                )
+                if rank_similarity
```

Comment:
maybe we can change this to `similarity_scale: List[str]` with a default of `[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]` or `[0,1]`